MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1148 @@
+---
+layout: post
+title: Editors in .NET MAUI DataForm(SfDataForm) control | Syncfusion
```

Comment:
Data Editors in .NET MAUI DataForm(SfDataForm) control


description: Learn about the supported data editors in Syncfusion .NET MAUI DataForm (SfDataForm) control in mobile and desktop applications from a single shared codebase.

Reply:
title and description updated